extensions/blocks/wordads/edit.js

```diff
@@ -4,27 +4,53 @@
 import { __ } from '@wordpress/i18n';
 import { BlockControls } from '@wordpress/editor';
 import { Component, Fragment } from '@wordpress/element';
-import { Placeholder, ToggleControl } from '@wordpress/components';
+import { ToggleControl } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
 import FormatPicker from './format-picker';
 import { AD_FORMATS } from './constants';
-import { icon, title } from './';
 
 import './editor.scss';
 
+/**
+ * Example images
+ */
+import rectangleExample from './example_300x250.png';
+import leaderboardExample from './example_728x90.png';
+import mobileLeaderboardExample from './example_320x50.png';
+import wideSkyscraperExample from './example_160x600.png';
+
 class WordAdsEdit extends Component {
 	handleHideMobileChange = hideMobile => {
 		this.props.setAttributes( { hideMobile: !! hideMobile } );
 	};
 
 	render() {
-		const { attributes, setAttributes } = this.props;
+		const { attributes, setAttributes, isSelected } = this.props;
 		const { format, hideMobile } = attributes;
 		const selectedFormatObject = AD_FORMATS.filter( ( { tag } ) => tag === format )[ 0 ];
-
+		const adControls = (
+			<ToggleControl
+				className="jetpack-wordads__mobile-visibility"
+				checked={ Boolean( hideMobile ) }
+				label={ __( 'Hide ad on mobile views', 'jetpack' ) }
+				onChange={ this.handleHideMobileChange }
+			/>
+		);
+		function getExampleAd( formatting ) {
+			switch ( formatting ) {
```

Comment:
@MichaelArestad should we add a `default` case for this switch? Not sure how likely/possible it is for there to be a value that is not covered in the current cases

Reply:
Feels like that could cover a bug somewhere in a way that it's hard to notice?

Reply:
TBH, I couldn't remember how to write the default (though I think it's just `default:`) and I defer to y'all.

Reply:
If deferring to me, then yes will you please add a default value?

Reply:
@dereksmart On it.